README.md

```diff
@@ -1,15 +1,42 @@
 # WELCOME TO LUNCH BUDDIES!!!
-### _An application that lets you find and match with like-minded students on campus to study, have a meal with, and more!_
----
+## Connect. Eat Lunch. Make Friends.
+
 # Table of Contents
-1. [Install](#install)
-2. [Usage](#usage)
-3. [Debugging](#debugging)
-4. [Contributing](#contributing)
+1[Description](#description)
+2[Install](#install)
+3[Usage](#usage)
+4[Debugging](#debugging)
+5[Contributing](#contributing)
     - [Front End](#front-end)
     - [Tips](#tips)
+6[License](#license)
```

Comment:
Is this change to the list intentional? Doesn't look as nice of a list to me imo